nodes/src/nodes/llm_openai_api/IGlobal.py

```
@@ -0,0 +1,96 @@
+            from openai import OpenAI, APIStatusError, OpenAIError, AuthenticationError, RateLimitError, APIConnectionError
+
+            # Get config
+            config = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+            apikey = config.get('apikey')
+            model = config.get('model')
+            base_url = config.get('base_url') or None
+            modelTotalTokens = config.get('modelTotalTokens')
+
+            # Only validate tokens > 0 if provided
+            if modelTotalTokens is not None:
+                if modelTotalTokens <= 0:
```

Ruff (0.15.7)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)